stl/debugger/STL.natvis

```diff
@@ -49,78 +49,56 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
       </Expand>
   </Type>
 
-
   <Type Name="std::tuple&lt;&gt;">
-      <DisplayString IncludeView="noparens"></DisplayString>
-      <DisplayString ExcludeView="noparens">({*this,view(noparens)})</DisplayString>
-      <Expand/>
-  </Type>
-
-  <Type Name="std::tuple&lt;*&gt;">
-      <DisplayString IncludeView="noparens">{_Myfirst._Val}</DisplayString>
-      <DisplayString ExcludeView="noparens">({*this,view(noparens)})</DisplayString>
-      <Expand>
-          <Item Name="[0]">_Myfirst._Val</Item>
-      </Expand>
-  </Type>
-
-  <Type Name="std::tuple&lt;*,*&gt;">
-      <DisplayString IncludeView="noparens">{_Myfirst._Val}, {((_Mybase *) this)->_Myfirst._Val}</DisplayString>
-      <DisplayString ExcludeView="noparens">({*this,view(noparens)})</DisplayString>
-      <Expand>
-          <Item Name="[0]">_Myfirst._Val</Item>
-          <Item Name="[1]">((_Mybase *) this)->_Myfirst._Val</Item>
-      </Expand>
-  </Type>
-
-  <Type Name="std::tuple&lt;*,*,*&gt;">
-      <DisplayString IncludeView="noparens">{_Myfirst._Val}, {((_Mybase *) this)->_Myfirst._Val}, {((_Mybase::_Mybase *) this)->_Myfirst._Val}</DisplayString>
-      <DisplayString ExcludeView="noparens">({*this,view(noparens)})</DisplayString>
-      <Expand>
-          <Item Name="[0]">_Myfirst._Val</Item>
-          <Item Name="[1]">((_Mybase *) this)->_Myfirst._Val</Item>
-          <Item Name="[2]">((_Mybase::_Mybase *) this)->_Myfirst._Val</Item>
-      </Expand>
+    <DisplayString IncludeView="noparens;noparensasptr"></DisplayString>
+    <DisplayString>({*this,view(noparens)})</DisplayString>
+    <Expand/>
   </Type>
 
-  <Type Name="std::tuple&lt;*,*,*,*&gt;">
-      <DisplayString IncludeView="noparens">{_Myfirst._Val}, {((_Mybase *) this)->_Myfirst._Val}, {((_Mybase::_Mybase *) this)->_Myfirst._Val}, {((_Mybase::_Mybase::_Mybase *) this)->_Myfirst._Val}</DisplayString>
-      <DisplayString ExcludeView="noparens">({*this,view(noparens)})</DisplayString>
-      <Expand>
-          <Item Name="[0]">_Myfirst._Val</Item>
-          <Item Name="[1]">((_Mybase *) this)->_Myfirst._Val</Item>
-          <Item Name="[2]">((_Mybase::_Mybase *) this)->_Myfirst._Val</Item>
-          <Item Name="[3]">((_Mybase::_Mybase::_Mybase *) this)->_Myfirst._Val</Item>
-      </Expand>
-  </Type>
+  <!--
+    Tuple uses a bit of a SFINAE-style hack in order to allow us to write less code.

```

Comment:
This comment is helpful, thanks! One nitpick: we conventionally avoid saying "hack" in product code. "Tuple uses a SFINAE-style technique" or something along those lines would be fine.